src/control/cmd/daos/util.go

```diff
@@ -69,6 +69,15 @@ func daosError(rc C.int) error {
 	return daos.Status(rc)
 }
 
+func dfsError(rc C.int) error {
```

Comment:
We have plenty of go code that works with error numbers, is there something dedicated needed here and why is it tied to dfs?

Reply:
yes this was only used in filesystem.go and now i use it in acl.go. so i moved it here. did not add something new.

Reply:
fyi you don't need to move it to this file to use it, you can still use the unexported function in any of the "daos" package's files

Reply:
got it, will try then. i thought i had an error when i tried it that way.